native/src/components/I18nProvider.tsx

```diff
@@ -46,7 +46,6 @@ const I18nProvider = ({ children }: I18nProviderProps): ReactElement | null => {
           /* Escaping is not needed for react apps:
              https://github.com/i18next/react-i18next/issues/277 */
         },
-        debug: buildConfig().featureFlags.developerFriendly,
```

Comment:
Just disables logging the whole i18n object every app start. If you would like to keep it let me know :) Just annoyed me during debugging this bc I did not see the logs.

Reply:
Good call

Reply:
like it as this also makes the tests much cleaner 😄